src/main/java/com/binance/connector/client/utils/WebSocketConnection.java

```diff
@@ -1,21 +1,18 @@
 package com.binance.connector.client.utils;
 
-import java.util.concurrent.atomic.AtomicInteger;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
-import okhttp3.WebSocket;
-import okhttp3.WebSocketListener;
+import okhttp3.*;
```

Comment:
Do not replace with *

Reply:
It was the Intellij that did it, not me.

Reply:
Please undo this.

Reply:
I'll do that one later today..